content/doc/book/pipeline/development.adoc

```diff
@@ -0,0 +1,145 @@
+---
+layout: section
+title: Pipeline Development Tools
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Pipeline Development Tools
+
+In this section, we'll discuss tools and resources
+that help speed development of Jenkins Pipelines.
+
+== Blue Ocean Editor (Declarative only)
+
+The
+<<../blueocean/pipeline-editor#, Blue Ocean Pipeline Editor>> provides a WYSIWYG
+way to create Declarative Pipelines. It gives you a structural view of all the stages,
```

Comment:
For clarity, I suggest removing the pronoun "It" for "The editor" and removing the "you/your" from the sentence, since this is reference material rather than a tutorial. For example: "The editor provides a structural view of all the stages, etc, etc in a Pipeline".